content/03.text-mining.md

```diff
@@ -12,9 +12,37 @@
 
 ### Text Mining for Relationship Extraction
 
-#### Rule-Based Natural Language Processing
+#### Rule-Based Relationship Extraction
 
-1. Mention papers on hand written rules and expressions
+Rule-based extraction consists of identifying sentences that contain important keywords or grammatical patterns that allude to relationships of interest. 
+These keywords are established via expert knowledge or pre-existing ontologies.
+Grammatical patterns are mainly constructed via experts curating part of speech tags or dependency path trees.
+Most text mining approaches use the above methods to generate features for machine learning algorithms and are discussed in later sections.
+For this section we focus on approaches that mainly use rule based extraction to detect sentences that assert a relationship.
+
+Rules that use grammatical patterns can be used to simplify sentences for easy extraction.
+One study used a set of grammar rules to reshape complex sentences with simpler versions.
+These simplified versions were manually curated to determine the presence of a relationship.
+By simplyfing sentences this approach achieved high recall, but had low precision [@pmid:21346999].
+Another approach used similar kind of simplification technique to make extraction easier [@doi:10.1093/database/baw156].
+Despite both successes, these kind of approaches rely heavily on manual curation and expert knowledge.
+This dependency raises concern for scalability as a lot of english sentences tend to fluctuate in their structure.
```

Comment:
I thought from what I read above that the simplification approach aimed to address exactly this. I was with you to this point, but then I got confused (or else I was wrong all along 😂 ).